### ISSUE TITLE
Multi class classification

### ISSUE THREAD

**Issue by Ganesh9100** [yubin-park/palobst#3]
**How to implement multi-class classification in this Palobst.**

from sklearn.datasets import make_hastie_10_2
from sklearn.model_selection import train_test_split

X, y = make_hastie_10_2(100)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)

from palobst import PaloBst

model = PaloBst(distribution="bernoulli")
model.warmup() # this runs JIT 
model.fit(X_train, y_train)
y_pred = model.predict_proba(X)[:,1]
# feature importances
print(model.feature_importances_)
